### PR TITLE
backport: 'trivial' pr14

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: doc/release-notes-19133.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## CLI
+
+A new `bitcoin-cli -generate` command, equivalent to RPC `generatenewaddress`
```
Comment (PastaPastaPasta) [inline doc/release-notes-19133.md:3]: drop 19357 unless we add 19133

File: doc/release-notes-18594.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## CLI
+
+The `bitcoin-cli -getinfo` command now displays the wallet name and balance for
```
Comment (PastaPastaPasta) [inline doc/release-notes-18594.md:3]: drop 19354 unless we do 18594

File: doc/release-notes-19133.md (hunk 1 above)
Comment (UdjinM6) [reply doc/release-notes-19133.md:3]: yeah, it's weird to have release notes for smth not yet implemented

File: doc/release-notes-18594.md (hunk 2 above)
Comment (UdjinM6) [reply doc/release-notes-18594.md:3]: same, agree

File: doc/REST-interface.md [hunk 3]
```diff
@@ -59,7 +59,6 @@ Only supports JSON as output format.
 * pruned : (boolean) if the blocks are subject to pruning
 * pruneheight : (numeric) highest block available
 * softforks : (array) status of softforks in progress
-* bip9_softforks : (object) status of BIP9 softforks in progress
```
Comment (UdjinM6) [inline doc/REST-interface.md:62]: 19390: too early for this, we still have `bip9_softforks`

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

**Comment by PastaPastaPasta** [comment]
dropped those three

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**